components/contextmenu/overview.md

```diff
@@ -155,6 +155,80 @@ A menu is often used to list pages, views or sections in an application so the u
 
 The same context menu can easily be attached to many targets, or you can use its `ShowAsync(x, y)` method to show it explicitly based on your business logic needs, data and events. Read more in the [Integration]({%slug contextmenu-integration%}) article.
 
+## Methods
+
+The ContextMenu methods are accessible through it's reference. You can use the available `HideAsync` method to programatically close the ContextMenu.
```

Comment:
```suggestion
The ContextMenu methods are accessible through it's reference:
* `ShowAsync` - programmatically shows the ContextMenu 
* `HideAsync`  - programatically hide the ContextMenu
```
I think it is valid to also list the ShowAsync method in this section, although it is commonly used to show the menu, it is still one of it methods